examples/capi-quick-start/nutanix-cluster-calico-helm-addon.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  labels:
+    ccm: nutanix
+    cluster.x-k8s.io/cluster-name: ${CLUSTER_NAME}
+    cluster.x-k8s.io/provider: infrastructure-nutanix
+  name: ${CLUSTER_NAME}
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+        - ${POD_CIDR:-192.168.0.0/16}
+    serviceDomain: cluster.local
+    services:
+      cidrBlocks:
+        - ${SERVICE_CIDR:-10.128.0.0/12}
+  topology:
+    class: nutanix-quick-start
+    controlPlane:
+      metadata: {}
+      replicas: ${CONTROL_PLANE_MACHINE_COUNT}
+    variables:
+      - name: clusterConfig
+        value:
+          addons:
+            cni:
+              provider: Calico
+              strategy: HelmAddon
+          controlPlane:
+            nutanix:
+              machineDetails:
+                additionalCategories:
+                  - key: key1
+                    value: value1
+                  - key: key2
+                    value: value2
+                bootType: legacy
+                cluster:
+                  name: ${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}
+                  type: name
+                gpus:
+                  - name: gpu1
+                    type: name
+                  - name: gpu2
+                    type: name
```

Comment:
Lets have just a basic example that will work without special hardware?

Reply:
removed.